<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height (%)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParHeight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>20.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParProminence">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prominence (%)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Background-to-Signal</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParSbr">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParSnr">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Noise-to-Signal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum peak height relative to max. data in that spectrum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum peak prominence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1 effectively means this criterion is unused</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower means less spectra are rejected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Height (%)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>For lowpass filter. Unknown interpretation</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParFitHeight">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99.000000000000000</double>
    </property>
    <property name="value">
     <double>25.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency cutoff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Gaussian fit function height</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParFreqCutoff">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.050000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum spectral intensity rel. to highest in dataset</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParIntThresh">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensity threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Gaussian fit function prominence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prominence (%)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParFitProminence">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>8.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="Filename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>681</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abs file path (waterflow.npz)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoAnalysis">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>791</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Background-prominence (%)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ParBackProm">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <double>99.000000000000000</double>
    </property>
    <property name="value">
     <double>6.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower means less spectra are rejected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoLogbook">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logbook</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Main author: Adrian Rutschmann (ETH/PSI). Co-authors Philipp Dijkstal and Sara Kjaer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
